spec/features/collection_drafts/forms/hidden_fields_spec.rb

```diff
@@ -0,0 +1,16 @@
+# MMT-1571
```

Comment:
we don't need the ticket number, this is something we've actually been trying to remove from our tests